content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+aliases: ["/news/contributor-interview-snor42"]
+title: "Freelance developper and Translator: meet SNor42"
+subtitle: "Discover the faces behind translations"
+date: 2023-12-06
+authors: [LéaDelin]
+icon: icon-world
+image: static/assets/images/2023/snor42.jpg
+tags:
+- contribute
+
+- community
+
+- contribution
+---
+**Have you ever wondered who’s hiding behind a Crowdin username? It’s time to find out! Contributor interviews are a great way of getting to know the people who participate in the PrestaShop project.**
+
+**For the second interview of 2023, we interview Norbert, aka [SNor42](https://crowdin.com/profile/snor42).**
+
+<img src="https://github.com/PrestaShop/prestashop.github.io/assets/113511191/5a649c1c-bb2c-465a-9e49-e1e4f737da67" width="384" height="512">
```

Comment:
it's not the best way to use photos in the article, I'll fix it after the merge

Reply:
Still not too sure how to add pictures, I'm experimenting everything at this point 🤪